cmd/cli/cmd/action/logs.go

```diff
@@ -0,0 +1,50 @@
+package action
```

Comment:
Please synchronize the docs to website, and make sure the change from `-c` to `-C` is applied onto our docs (assuming we used this flag somewhere in the documents). Also, replace `argo logs` usage with `capact act logs` in all docs. Thanks!